File: angular-rxjs/frontend/src/app/song.ts
export class Song {
  id: number;
  artist: string;
  title: string;
  length: number; // másodpercben
  likes: number;

  constructor(id: number, artist: string, title: string, length: number, likes: number) {
    this.id = id;
    this.artist = artist;
    this.title = title;
    this.length = length;
    this.likes = likes;
  }
}

File: angular-rxjs/backend/angular-rxjs/angular-rxjs/Models/Song.cs
namespace angular_rxjs.Models
{
    public class Song
    {
        public int Id { get; set; }
        public string Artist { get; set; } = string.Empty;
        public string Title { get; set; } = string.Empty;
        public int Length { get; set; } // másodpercben
        public int Likes { get; set; }
        public string Link { get; set; } = string.Empty;
    }
}
